src/main/java/seedu/programmer/model/student/LabResult.java

```diff
@@ -1,28 +1,40 @@
 package seedu.programmer.model.student;
 
 import static java.util.Objects.requireNonNull;
+import static seedu.programmer.commons.util.AppUtil.checkArgument;
 
 /**
  * Represents a student's name in the ProgrammerError.
  */
 public class LabResult {
+    public static final String MESSAGE_CONSTRAINT =
+            " Lab result should be a non-negative number or a unmarked placeholder(-1)";
```

Comment:
Similarly for this one, it should refer to the constraint in Lab class.

Reply:
Ok have changed in the latest commit!